graphql-examples/snowtooth/query.bal

```diff
@@ -53,7 +53,7 @@ service /graphql on new graphql:Listener(9000) {
         if array:length(lifts) > 0 {
             return new Lift(lifts[0]);
         } else {
-            return ();
+            return;
```

Comment:
We can remove the `else` part altogether here, right?

Reply:
Duh! fixed it.